storage/src/testFixtures/java/tech/pegasys/teku/storage/client/ChainUpdater.java

```diff
@@ -152,7 +152,7 @@ public SignedBlockAndState finalizeCurrentChain() {
       saveBlock(newChainHead);
       updateBestBlock(newChainHead);
     }
-    final Checkpoint finalizedCheckpoint = newChainHead.getState().getFinalized_checkpoint();
+    final Checkpoint finalizedCheckpoint = newChainHead.getState().getFinalizedCheckpoint();
```

Comment:
GitHub code scanning correctly pointed out that `newChainHead` _could_ be null. But this is just a test fixture and those values are hardcoded, so it doesn't matter.

Reply:
Fortunately it has "Used in tests" and "Won't fix" options to make it be quiet. :)